src/platform_impl/ios/monitor.rs

```diff
@@ -18,31 +18,44 @@ pub struct VideoMode {
     pub(crate) size: (u32, u32),
     pub(crate) bit_depth: u16,
     pub(crate) refresh_rate: u16,
-    pub(crate) screen_mode: id,
+    pub(crate) screen_mode: NativeDisplayMode,
     pub(crate) monitor: MonitorHandle,
 }
 
+#[derive(Debug, PartialEq, Eq, Hash)]
+pub struct NativeDisplayMode(pub id);
+
+unsafe impl Send for NativeDisplayMode {}
+
+impl Drop for NativeDisplayMode {
+    fn drop(&mut self) {
+        unsafe {
+            let () = msg_send![self.0, release];
```

Comment:
@aleksijuvani @simlay I was just merging master back into my `winit-glutin-next` and noticed that the `assert_main_thread` has been removed. Is this intentional?

Reply:
I think it's fine to pass this around multiple threads. If we see a crash here we'll revisit.